src/main/scala/org/zalando/nakadi/client/actor/ScoopListenerActor.scala

```diff
@@ -87,11 +88,11 @@ class ScoopListenerActor(listener: Listener,
   def handleMemberUnreachable(member: Member): Unit = {
     currentCluster match {
       case Some(actualCluster) =>
-        if(actualCluster.selfRoles.contains("leader")) { // TODO right role name?
+        if(actualCluster.selfRoles.contains("leader")) { // TODO right role name?   // config?
```

Comment:
hmm..not sure..I think it is unlikely the Akka cluster role name changes frequently